Source/AFError.swift

```diff
@@ -139,6 +139,8 @@ public enum AFError: Error {
         case decodingFailed(error: Error)
         /// Generic serialization failed for an empty response that wasn't type `Empty` but instead the associated type.
         case invalidEmptyResponse(type: String)
+        /// The response serializer was added to the request after the request was already completed.
```

Comment:
I think this should say "A response serializer...", and "completed" should probably be "finished."

Reply:
Resolved in 656f331d.